docs/wallet/wallet-sync.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Wallet Sync
+id: wallet-sync
+---
+
+## High level wallet state
+
+Our wallet infrastructure has a specific table called `state_descriptors`.
+This tracks chain state for our wallet.
+Here is an example of the contents of this table
+
+>sqlite> select * from state_descriptors;
+SyncHeight|0000000000000000000134aa9e949ea1d053042b8dfa59bdc73b0322a88f009e 665741
+
+If you look carefully in the second column, you will see a string encoding indicating
+what the wallet state is. In this case, the last block hash seen by the wallet is
+
+>0000000000000000000134aa9e949ea1d053042b8dfa59bdc73b0322a88f009e
+
+and height
+
+>665741
+
+Alternatively, you can retrieve this information with `bitcoin-s-cli`
+
+```
+ ./bitcoin-s-cli walletinfo
+{
+  "wallet": {
+    "keymanager": {
+      "rootXpub": "..."
+    },
+    "xpub": "...",
+    "hdPath": "m/84'/0'/0'"
+  }
+}
+```
```

Comment:
imo this doesn't really make sense to include here. If you are using bitcoin-s as a library then you won't have access to the cli/server

Reply:
I really liked this section on the rescan docs, i was trying to replicate it here 

https://bitcoin-s.org/docs/next/wallet/wallet-rescan#rescan-from-cli

maybe I should have an appropriate heading like we do there? 

>Wallet state from the cli

or something of that nature?

Reply:
Yeah that would make more sense